packages/react-components/react-aria/src/AriaLiveAnnouncer/useAriaNotifyAnnounce.ts

```diff
@@ -0,0 +1,46 @@
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import * as React from 'react';
+
+import type { AriaLiveAnnounceFn } from './AriaLiveAnnouncer.types';
+
+type AriaNotifyOptions = {
+  notificationID?: string;
+  priority?: 'none' | 'important';
+  interrupt?: 'all' | 'pending' | 'none';
+};
+
+type DocumentWithAriaNotify = Document & {
+  ariaNotify: (message: string, options: AriaNotifyOptions) => void;
+};
+
+/* INTERNAL: implementation of the announcer using the ariaNotify API */
+export const useAriaNotifyAnnounce_unstable = (): AriaLiveAnnounceFn => {
+  const { targetDocument } = useFluent();
+
+  const announce: AriaLiveAnnounceFn = React.useMemo(
```

Comment:
Nit: Should this be `useCallback` rather than `useMemo`?

Reply:
haha possibly. I was wondering that too and ended up keeping `useMemo` just because that was what was currently being used (in `useAriaLiveAnnouncer, in the code I deleted and moved over to `useDomAnnounce`).

I don't really have an opinion one way or the other, I'm happy to change it